core/src/packets/icmp/v6/mod.rs

```diff
@@ -280,18 +317,20 @@ impl Icmpv6Payload for () {
 
 /// A trait for common behaviors shared by ICMPv6 packets.
 ///
-/// For convenience, use the `Icmpv6Packet` derive macro on Icmpv6 Payloads,
+/// ## Derivable
+///
+/// The `Icmpv6Packet` trait can be used with `#[derive]` on Icmpv6 payloads,
 /// which also derives the implementation for the `Packet` trait.
 ///
-/// # Example
 /// ```
 /// #[derive(Icmpv6Packet)]
 /// pub struct EchoReply {
 ///     ...
 /// }
 /// ```
 ///
-/// # Remarks
+/// ### Remarks
```

Comment:
I think we should keep `Remarks` at the same heading level as `Derivable`.

Reply:
ok